rpcq/core-messages.py

```diff
@@ -13,8 +13,8 @@
 @dataclass(eq=False, repr=False)
 class Frame(Message):
     """
-    A frame encapsulates any rotating frame
-relative to which control or readout waveforms may be defined.
+    A frame encapsulates any rotating frame ~
```

Comment:
This is a little odd. The `~\n` special syntax for Lisp is supposed to be consumed so that it emits `"""A frame encapsulates any rotating frame relative to which contorl or readout waveforms may be defined."""`. There's a bug somewhere in part of the python code generator responsible for writing out these docstrings, though I don't imagine you introduced it.

Reply:
Oh: the `~\n` syntax is probably special to format strings, not to the Lisp reader and not to data strings that get substituted into format string directives like `~a`.